conformance/reports/v1.0.0/nginxinc-nginx-gateway-fabric/README.md

```diff
@@ -1,9 +1,10 @@
-# Nginxnc Nginx Network Fabric
+# Nginxinc NGINX Gateway Fabric
 
 ## Table of Contents
 
 |API channel|Implementation version|Mode|Report|
 |-----------|----------------------|----|------|
 |x|[v1.1.0](https://github.com/nginxinc/nginx-gateway-fabric/releases/tag/v1.1.0)|x|[link](./v1.1.0-report.yaml)|
+|x|[v1.2.0](https://github.com/nginxinc/nginx-gateway-fabric/releases/tag/v1.2.0)|x|[link](./v1.2.0-report.yaml)|
 
 ## Reproduce
```

Comment:
Sorry we missed this one @sjberman! Do you mind adding some steps for how to reproduce these results? Example here: https://github.com/kubernetes-sigs/gateway-api/tree/main/conformance/reports/v1.0.0/kong-kubernetes-ingress-controller